src/SFA.DAS.AssessorService.Web/Views/Application/Standard/ConfirmStandard.cshtml

```diff
@@ -75,12 +99,12 @@
                             <div class="govuk-checkboxes__item">
                                 <input class="govuk-checkboxes__input" asp-for="@Model.IsConfirmed" name="IsConfirmed" type="checkbox">
                                 <label class="govuk-label govuk-checkboxes__label" for="IsConfirmed">
-                                    By continuing with this application you are confirming you have <a href="@Model.SelectedStandard.StandardData.AssessmentPlanUrl" target="_blank" class="govuk-link">read the assessment plan</a> (PDF).
+                                    By continuing, you confirm you have read and understood the relevant assessment plan(s) for <a href="@Model.SelectedStandard?.StandardPageUrl" target="_blank" class="govuk-link">this standard.</a>
```

Comment:
This is probably a quirk of the HTML, but when testing in IE, I wasn't able to click the link to view the standard page url. It instead ticked/unticked the tickbox, I'm assuming it's becaue it's wrapped in the label....

Reply:
The url import was missing from the PR hence the link wasnt woking. Fixed now.